src/components/Picker/BasePicker/index.js

```diff
@@ -11,6 +11,25 @@ class BasePicker extends React.Component {
         super(props);
 
         this.executeOnCloseAndOnBlur = this.executeOnCloseAndOnBlur.bind(this);
+        this.setDefaultValue = this.setDefaultValue.bind(this);
+    }
+
+    componentDidMount() {
+        this.setDefaultValue();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (prevProps.items === this.props.items) {
+            return;
+        }
+        this.setDefaultValue();
+    }
+
+    setDefaultValue() {
```

Comment:
We only set Default Value if number of items is one. So, can you please add a comment mentioning it.